Main/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+.env
+.DS_Store
```

Comment:
I think we should add `package-lock.json` to this list!

Reply:
Good idea, I have added this to my latest PR. thank you :)

Reply:
I just did a quick google to check, and the package-lock.json is supposed to be in the git repo, so this ensures we are all using the same node modules/ dependencies versions. I will remove my latest PR including it in the gitignore. thanks! here is the link in you're interested: https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5